### PR TITLE
764 update mng01

### LINKED ISSUE #764: update mng01
`g_lineplot` has new arguments to control line type. Expose it in `mng01_main`

### PR DESCRIPTION
**Pull request by BFalquet**
close #764 

add `line_type` argument to control line type in mng01

thank you for the review

### CODE REVIEW

File: R/mng01.R [hunk 1]
```diff
@@ -74,7 +78,6 @@ mng01_main <- function(adam_db,
   assert_subset(table, c("n", center_fun_choice, interval_fun_choice))
 
   df <- adam_db[[dataset]]
-  line_col <- unlist(line_col)
```
Comment (Melkiades) [inline R/mng01.R:77]: why unlist?

File: R/mng01.R [hunk 2]
```diff
@@ -103,20 +106,37 @@ mng01_main <- function(adam_db,
     subject_var = "USUBJID"
   )
 
-  if (!is.null(names(line_col))) {
-    color_lvl <- sort(unique(df[[arm_var]]))
-    col <- line_col[as.character(color_lvl)]
+
+  arm_lvl <- sort(unique(df[[arm_var]]))
+  col <- if (!is.null(names(line_col))) {
+    col <- line_col[as.character(arm_lvl)]
```
Comment (Melkiades) [inline R/mng01.R:112]: I know it works anyway but to avoid confusion can you use a different name for the internal var?

**Review by Melkiades — APPROVED** [review]
Lgtm! Thanks Benoit :)))

File: R/mng01.R (hunk 1 above)
Comment (BFalquet) [reply R/mng01.R:77]: I think we were accepting list before (now we are asserting for character) and we forgot to remove that bit.

File: R/mng01.R (hunk 2 above)
Comment (BFalquet) [reply R/mng01.R:112]: good point, let me correct that.

**Pull request merged**